content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -41,12 +41,12 @@ cluster's API server.
 
 ## Define clusters, users, and contexts
 
-Suppose you have two clusters, one for development work and one for scratch work.
+Suppose you have two clusters, one for development work and one for test work.
```

Comment:
@reylejano I had a go at making the required changes. I'm open to any suggestions and would be grateful for any feedback you can offer.